packages/server/app/Controllers/Http/AuthenticationController.ts

```diff
@@ -185,4 +188,38 @@ export default class AuthenticationController {
             status: 'SUCCESS',
         };
     }
+
+    public async confirmEmail({
+        request,
+        response,
+        auth,
+        bouncer,
+    }: HttpContextContract): Promise<ConfirmEmailResponseBody> {
+        const user = auth.user;
+        invariant(
+            user !== undefined,
+            'User must be authenticated to get her profile information',
+        );
+
+        await bouncer.authorize('confirmEmail');
```

Comment:
Thoughts: we will have to apply this bouncer to every authenticated routes don't we ?

Reply:
Not this bouncer exactly, because this one ensures the user **has not** confirmed her email. But, indeed, we will have to implement a bouncer that ensures the user **has** confirmed her email for every authenticated route!